security-ldap/src/test/groovy/io/micronaut/security/ldap/LdapAuthenticationSpec.groovy

```diff
@@ -200,9 +200,10 @@ class LdapAuthenticationSpec extends InMemoryLdapSpec {
 
         when:
         LdapAuthenticationProvider authenticationProvider = ctx.getBean(LdapAuthenticationProvider)
-        Publisher<AuthenticationResponse> response = authenticationProvider.authenticate(null, createAuthenticationRequest("euclid", "abc"))
-        TestSubscriber subscriber = Flowable.fromPublisher(response).test().assertError(AuthenticationException)
-        List<Throwable> throwableList =subscriber.errors()
+        TestSubscriber subscriber = new TestSubscriber<>()
+        Flowable.fromPublisher(authenticationProvider.authenticate(null, createAuthenticationRequest("euclid", "abc"))).blockingSubscribe(subscriber)
+        subscriber.assertError(AuthenticationException)
+        List<Throwable> throwableList = subscriber.errors()
```

Comment:
Is this just preference or is there a benefit to this change?

Reply:
Tests fail without this change.